CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 # PayPal iOS SDK Release Notes
 
 ## unreleased
+* PayPalUI
+    * Separate buttons into `UIKit` and `SwiftUI` implementations.
```

Comment:
Can we add a bullet here for the iOS 16 bug fix as well? Something like "Fixes fatal error crash with iOS 16 targets".

Reply:
I think it would be helpful to call out the specific crash too `UIViewRepresentables must be value types`